packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -60,10 +69,7 @@ const CommerceApiProvider = (props: CommerceApiProviderProps): ReactElement => {
         throwOnBadResponse: true
```

Comment:
Non-blocking small comment, Isomorphic allows a `fetchOptions` in the config, we may want to allow it as a prop for the Provider here. 
```suggestion
        throwOnBadResponse: true,
        fetchOptions,
```

Reply:
✅